react/src/hooks/SelectedComponents.js

```diff
@@ -22,9 +22,11 @@ const useSelectedComponents = (componentDefinitions) => {
             const index = this.__lookup(type, name);
             if (index >= 0) {
                 this.__unselect(type, name, index);
+                return false;
```

Comment:
If this is going to return a value, should select and unselect also return a value? I'm not sure. Maybe not. But I figured I'd ask.